dashboard/config/levels/custom/aichat/AI Chat Test Level.level

```diff
@@ -10,7 +10,8 @@
     "long_instructions": "## Sample Instructions\r\n\r\nThis is a chat bot which allows you to talk to historical figures.\r\n\r\n### Do This\r\n\r\nType in a question and press the `Submit`(#7435b2) button. What can you learn about Ada Lovelace?",
     "system_prompt": "You are a chatbot for a middle school classroom where they can chat with a historical figure. You must answer only questions about the computer science and computing-related topics. You will act as Ada Lovelace; every question you answer must be from her perspective. Wait for the student to ask a question before responding.",
     "bot_title": "HistoryBot",
-    "bot_description": "This is a bot which will simulate the interaction with a historical figure."
+    "bot_description": "This is a bot which will simulate the interaction with a historical figure.",
+    "disable_projects": true
```

Comment:
Could you add this to `AI Chat Test Level - 2.level`?

Reply:
Ack didn't know that existed -- updated! In case you ever run into this, there was some recent work that [moved around our level configuration files](https://github.com/code-dot-org/code-dot-org/pull/53066), and our two aichat levels are in different directories at the moment.

Reply:
I don't, really. I believe the DSL part is "domain-specific language", generally described [here](https://martinfowler.com/dsl.html), but I don't know the distinction between "DSL-defined" and "custom" levels. Sorry to not be more helpful!